scanner.go

```diff
@@ -483,7 +483,7 @@ func stateInStringSingle(s *scanner, c byte) int {
 func stateInStringEsc(resume func(s *scanner, c byte) int) func(s *scanner, c byte) int {
 	return func(s *scanner, c byte) int {
 		switch c {
-		case 'b', 'f', 'n', 'r', 't', '\\', '/', '"', '\'', '\n':
+		case 'b', 'f', 'n', 'r', 't', '\\', '/', '"', '\'', '\n', 'x':
```

Comment:
This doesn't seem right. Unlike `\n` or `\t` (and like `\u`), `\x` does have some special state after it.

Looks like need 2 extra states `stateInStringEscX` and `stateInStringEscX1` similar to the 4 states `stateInStringEscU*`

Reply:
BTW to see that this in a real bug, specify an incorrect raw binding ending with `\x`. Result: micro doesn't report any error, it only silently refuses to register this keybinding.

In the fixed version with `stateInStringEscX` and `stateInStringEscX1`, micro prints an error as expected.

Reply:
Ok, thank you very much for that hint!
It's definitely not intended to cause regressions.

As you already pointed out at a different location:
`micro` and his dependencies are far away from being simple. :wink: